index.html

```diff
@@ -63,6 +63,18 @@ <h1>link2aws.github.io</h1>
 
             //document.getElementById("text").value = "arn:aws:iam::aws:policy/AdministratorAccess";
             onUpdate();
+
+            // If the user passes in the arn as a parameter, immediately redirect them
+            var params = new URLSearchParams(window.location.search);
```

Comment:
Would you be ok with switching to window.location.hash?

I am asking for this change because this way, the ARN will not be sent to the web server. This is better for privacy.

Please let me know if you see any reason against window.location.hash. If there are any technical problems with it, then we can keep window.location.search. But if they are equal then I would prefer the more privacy friendly option.